Dalamud/Game/Config/GameConfig.cs

```diff
@@ -0,0 +1,450 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+using Dalamud.IoC;
+using Dalamud.IoC.Internal;
+using Dalamud.Memory;
+
+using FFXIVClientStructs.FFXIV.Common.Configuration;
+using Serilog;
+
+namespace Dalamud.Game.Config;
+
+/// <summary>
+/// This class represents the game's configuration.
+/// </summary>
+[InterfaceVersion("1.0")]
+[PluginInterface]
+[ServiceManager.BlockingEarlyLoadedService]
```

Comment:
Is there a reason this is BlockingEarlyLoadedService instead of a regular EarlyLoadedService?

Reply:
because every other service was and I have no idea how any of this works